src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,24 @@
+import calculate from '../logic/calculate';
+
+describe('Test of all calculate functions', () => {
+  test('will return 4 by adding 2 + 2', () => {
+    const myOb = { total: '2', next: '2', operation: '+' };
+    const resObj = { total: '4', next: null, operation: null };
+    expect(calculate(myOb, '=')).toEqual(resObj);
+  });
+  test('will return the value of 5 by substract 8 - 3', () => {
+    const myObj = { total: '8', next: '3', operation: '-' };
+    const ResObj = { total: '5', next: null, operation: null };
+    expect(calculate(myObj, '=')).toEqual(ResObj);
+  });
+  test('will return the value of null by pressing AC', () => {
+    const myObj = { total: '2', next: '2', operation: '÷' };
+    const ResObj = { total: '0', next: null, operation: null };
+    expect(calculate(myObj, 'AC')).toEqual(ResObj);
+  });
+  test('will return the value of 4 by multiply 3 x 2', () => {
+    const myObj = { total: '3', next: '2', operation: 'x' };
+    const ResObj = { total: '6', next: null, operation: null };
+    expect(calculate(myObj, '=')).toEqual(ResObj);
+  });
+});
```

Comment:
- [ ] As required for the project all the functionalities & scenarios of the `calculate.js` module to be covered using Jest, kindly make sure you add tests scenario to `'+/-'`,  `decimal numbers` (i.e. numbers including `.` e.g. `3.5`, `4.23`,...), etc.